app/src/main/java/com/duckduckgo/app/featureusage/FeatureSegmentsManagerImpl.kt

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
As with the other comment, everything in this package is specific to the segments, not sure why it has to exist in the app module?

Reply:
I wouldn't create a module because it's going to be removed quite soon but probably makes sense to move it to statistics as everything is related to set and send the new pixel 👍